src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,106 @@
-import movies from '../../api/movies.json';
-
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
+import { getMovie } from '../../api/api';
+
+export const FindMovie = ({ addMovie }) => {
+  const [title, setTitle] = useState('');
+  const [errorMessage, setErrorMessage] = useState('');
+  const [movieNew, setMovieNew] = useState({});
+  const [isMovieFound, setIsMovieFound] = useState(false);
+
+  const inputTitle = (value) => {
+    setTitle(value);
+    setErrorMessage('');
+    setIsMovieFound(false);
+  };
+
+  const handleResult = async() => {
+    getMovie(title)
+      .then((movieFromServer) => {
+        if (movieFromServer.Title) {
+          setMovieNew({ ...movieFromServer });
+          setIsMovieFound(true);
+        } else {
+          setErrorMessage(`Can't find a movie with such a title`);
+        }
+      });
+  };
+
+  const submitResult = () => {
+    addMovie(movieNew);
+    setMovieNew({});
+    setTitle('');
+    setIsMovieFound(false);
+  };
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
-
-      <div className="field is-grouped">
-        <div className="control">
-          <button
-            type="button"
-            className="button is-light"
-          >
-            Find a movie
-          </button>
+  return (
+    <>
+      <form
+        className="find-movie"
+        onSubmit={(event) => {
+          event.preventDefault();
+          submitResult();
+        }}
+      >
+        <div className="field">
+          <label className="label" htmlFor="movie-title">
+            Movie title
+          </label>
+
+          <div className="control">
+            <input
+              type="text"
+              id="movie-title"
+              placeholder="Enter a title to search"
+              className={classnames('input', { 'is-danger': !isMovieFound })}
+              value={title}
+              onChange={event => inputTitle(event.target.value)}
+            />
+          </div>
+          {errorMessage ? (
+            <p className="help is-danger">
+              {errorMessage}
+            </p>
+          ) : ''}
         </div>
 
-        <div className="control">
-          <button
-            type="button"
-            className="button is-primary"
-          >
-            Add to the list
-          </button>
+        <div className="field is-grouped">
+          <div className="control">
+            <button
+              type="button"
+              className="button is-light"
+              onClick={handleResult}
```

Comment:
```suggestion
              onClick={findMovie}
```
handleResult and submitResult sound very close to each other and it's hard to get what each function does.